### DISCUSSION TITLE
Welcome to chatify Discussions!

### DISCUSSION THREAD

**Discussion by munafio** [munafio/chatify#192] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pritborteam** [comment]
Can this be used with Laravel 11? I see requirement mentions  <11.

**Reply by GorchPortley** [reply]
i am using it successfully with laravel 11, no issues with install at all, just have to write rules in your database.php config or set strict to false.
